toaster_msgs/srv/PlotFactsDB.srv

```diff
@@ -0,0 +1,7 @@
+string subjectID
+string targetID
+string a
```

Comment:
please put a more specific name than a and b.
Plus if it's time you shouldn't use string...